recipes/digen/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "digen" %}
+{% set version = "0.0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/digen-{{ version }}.tar.gz
+  sha256: ca8c7b27bd15ce4021d0ae8abce64e87be3e8717e7e40fe75ee5db0a1de79c2b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

Comment:
It is available. Moreover, it is detected as dependency and attempted to be installed.
I am not sure why this installation fails and results with the following error:
` digen 0.0.5 requires xgboost, which is not installed.`
This applies to Win64 only. No issues on Linux, or OS X.

Reply:
Looks like [one of the dependencies is not available for Windows](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=486391&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b&l=1564). Our policy is that for packages that could be noarch, but are missing dependencies on a platform, we still ship them as noarch because the dependencies could become available in the future.

Reply:
Sounds good! Does it mean the package is ready to be merged?
And is there an option to add `colcon-argcomplete` as pip dependency?

Reply:
Would you rather see this line 
`   skip: true  # [win] `
substituted with that one?
`noarch: python`
This will end up in win64 test failure.

Reply:
If conda says it installs `xgboost` and then `pip check` says that `xgboost` is not installed, then that is a problem with the `xgboost` package or `pip` not with your package. You can always test the noarch package on a Windows machine to see if it works.

Reply:
It doesn't work, I get the same error on Windows. 
So the choice is changing to 27c922286a3fd48caa52f13dedd84bdddd7a402d and failing on Win64, or skipping Win64 tests, whatever works better for you @carterbox .

Reply:
The only way I am able to install xgboost on Windows is using `pip` from the anaconda bash.

Reply:
> And is there an option to add colcon-argcomplete as pip dependency?

No, if your package has dependencies that are not on conda-forge, you need to add them to conda-forge by adding a recipe for them in this pull-request.

> The only way I am able to install xgboost on Windows is using pip from the anaconda bash.

You should only be installing dependencies using conda. If there are missing dependencies for you package, you need to add a recipe for them in this pull-request.

> It doesn't work, I get the same error on Windows.

As I tried to explain above, the problem is with `pip check` or `xgboost`. The import tests pass, so if your package works as expected, it doesn't matter if `pip check` can't find `xgboost`.

Reply:
Thank you for your explanation. I have applied the requested changes. Let's stay on `noarch`.